### PR TITLE
refine the encoding of the arabic examples

### PR DESCRIPTION
**Pull request by MedKhem**

### CODE REVIEW

File: Schemas/TEILex0/TEILex0.examples/examples.xml [hunk 1]
```diff
@@ -835,114 +837,335 @@
...
+                        <gram type="case" resp="khemakhem" value="accusative"/>
+                     </gramGrp>
+                     <orth>أُُُرُوكًا</orth>
+                  </form>
+                  <sense xml:id="ark1_1_sense1">
+                     <cit type="example">
+                        <quote>الجملُ</quote>
+                     </cit><pc>:</pc> 
+                     <def>اكل ورق الأَرَاك</def>
+                     ِِ</sense>
```
Comment (ttasovac) [inline Schemas/TEILex0/TEILex0.examples/examples.xml:866]: @MedKhem can you check this —  you have a kasra (according to my Unicode utility, I am not trying to pretend I know any Arabic letters :) outside the def. we can't have text content inside sense.

File: Schemas/TEILex0/TEILex0.examples/examples.xml [hunk 2]
```diff
@@ -835,114 +837,335 @@
...
+                        <gram type="number" value="plural">ج </gram>
+                     </gramGrp>
+                     <form type="variant"><orth>أُرُك </orth></form>
+                     <lbl>و</lbl>
+                     <form type="variant"><orth>أَرَائِك</orth></form>
+                  </form>
+                  <sense xml:id="ark1_3_sense1">
+                     <def>شجر ذو شوك. طويل الساق، كثير الورق و الأغصان. خوَّار العود تتَّخذ منه
+                        المساويكِ</def>
+                     ِِ</sense>
```
Comment (ttasovac) [inline Schemas/TEILex0/TEILex0.examples/examples.xml:914]: kasra

File: Schemas/TEILex0/TEILex0.examples/examples.xml [hunk 3]
```diff
@@ -835,114 +837,335 @@
...
+                  <form type="inflected">
+                     <gramGrp>
+                        <gram type="case" resp="khemakhem" value="accusative"/>
+                     </gramGrp>
+                     <orth>أُرَكًا</orth>
+                  </form>
+                  <sense xml:id="ark2_1_sense1"> 
+                     <cit type="example"><quote>الجرحُ</quote></cit><pc>:</pc> 
+                     <def>برأَ</def>
+                     ِِ</sense>
```
Comment (ttasovac) [inline Schemas/TEILex0/TEILex0.examples/examples.xml:949]: kasra

File: Schemas/TEILex0/TEILex0.examples/examples.xml (hunk 1 above)
Comment (MedKhem) [reply Schemas/TEILex0/TEILex0.examples/examples.xml]: well seen +1 . I fixed that in my latest commit

**Pull request merged**